### ISSUE TITLE
Storage permission not requested for local content directory server

### ISSUE THREAD

**Issue by martynsmall** [trishika/DroidUPnP#59]
For the local content directory to work the app needs the storage permission, and from Marshmallow onwards this isn't granted when the app is installed.  It should be requested at runtime when required, however it isn't and as such when you open any of the three media categories they appear empty.  The app needs to check if this permission is available and request it from the user if not.

**Comment by Strubbl** [comment]
Yes, workaround is to go to settings app --> apps --> droidupnpn --> permissions --> activate storage permission
